concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -42,7 +42,7 @@ def upload_new_file_on_cluster(task_id = '0', part_id = '0', current_time = 0):
     file_content    = task_id
```

Comment:
I think it would be better to remove defaults `task_id` and `part_id` here. I think that in most cases we'll need to pass a value matching the task anyway. And having a default makes it very easy to forget that you have to set this.

Reply:
done :+1: